smoke-tests/apps/TraceLog4j2UsingAgent/src/smokeTest/resources/applicationinsights.json

```diff
@@ -10,5 +10,8 @@
     "logging": {
       "level": "warn"
     }
+  },
+  "preview": {
+    "captureLog4jMarkerAttribute": true
```

Comment:
The feature only works with Log4j 2 (not for Log4j 1). Would `captureLog4j2MarkerAttribute` be a better property name?

Reply:
@trask @heyams This PR could be tested and merged after the next `opentelemetry-java-instrumentation` release. Early feedbacks are welcome.